codegen/build-oas.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+set -eux -o pipefail
+
+version=$1 # e.g. 2024-07
+modules=("control" "data")
+
+destination="src/pinecone-generated-ts-fetch"
+build_dir="build"
+
+update_apis_repo() {
+	echo "Updating apis repo"
+	pushd codegen/apis
+		git fetch
+		git pull
+		just build
```

Comment:
Leaving this in seems to cause failures when running locally:

```
npm run generate:openapi

> @pinecone-database/pinecone@2.2.2 generate:openapi
> ./codegen/build-oas.sh 2024-07 && npm run build && npm run format

+ version=2024-07
+ modules=("control" "data")
+ destination=src/pinecone-generated-ts-fetch
+ build_dir=build
+ update_apis_repo
+ echo 'Updating apis repo'
Updating apis repo
+ pushd codegen/apis
~/workspace/pinecone-ts-client/codegen/apis ~/workspace/pinecone-ts-client
+ git fetch
+ git pull
Already up to date.
+ just build
error: No justfile found
```

Is the just command still needed here?

Reply:
I'm not sure if this is relevant here, but I had to do this when calling justfile commands in the submodule in a parent repo:

`just -f apis/Justfile build`